peps/pep-9999.rst

```diff
@@ -286,13 +299,52 @@ Some examples from the conda-forge mapping include:
 
 The mappings MAY also specify another section ``package_managers``, reporting
 which package managers are available in the ecosystem. This field MUST
-take a dictionary with the following fields:
+take a list of dictionaries, with each of them reporting the following fields:
 
-- ``name`` (string), usually the name of the package manager executable
+- ``name`` (string), usually the name of the package manager executable.
 - ``install_command`` (list of strings), the command to run to install the mapped package(s).
+- ``query_command`` (list of strings), the command to check if the mapped package(s)
+  are already installed.
+- ``requires_elevation`` (bool, ``install`` or ``query``): whether the associated commands require
+  superuser permissions to run.
 - ``version_operators``: a mapping of PEP 440 operator names to the relevant
   syntax for this package manager.
 
+
+Governance
+----------
+
+The central registry and the list of known ecosystems MUST be maintained by a
+central authority. These responsibilities include:
+
+- Guaranteeing access to the central registry and list of known ecosystems
+  documents, via online and offline (prepackaged) mechanisms.
+- Maintaining the central registry up-to-date: curating and triaging requests
```

Comment:
```suggestion
- Keeping the central registry up-to-date: curating and triaging requests
```

"Maintaining … up-to-date" sounds wrong to me, and it took me a few attempts to actually comprehend this sentence.